src/context/EarnManagerProvider/providers/yearn/api/api.ts

```diff
@@ -162,8 +174,15 @@ export class YearnVaultApi {
   }
 
   async withdraw(input: WithdrawInput): Promise<string> {
-    const { amountDesired, bip32Params, dryRun = false, vaultAddress, userAddress, wallet } = input
-    if (!wallet || !bip32Params || !vaultAddress) throw new Error('Missing inputs')
+    const {
+      amountDesired,
+      accountNumber = 0,
+      dryRun = false,
+      vaultAddress,
+      userAddress,
+      wallet
+    } = input
+    if (!wallet || !vaultAddress) throw new Error('Missing inputs')
     const estimatedGas: BigNumber = await this.withdrawEstimatedGas(input)
```

Comment:
there is no try/catch around this async function. Are you sure you want to bubble the error up to the caller to handle?

Reply:
Having the error bubble up to the client, so the UI can handle errors with on chain interactions more gracefully made sense to us.